snippets/take.md

```diff
@@ -0,0 +1,10 @@
+### Take
+
+Use `.slice()` to create a slice of the array with n elements taken from the beginning.
+
+```js
+const take = (arr, n) => n === undefined ? arr.slice(0, 1) : arr.slice(0, n);
```

Comment:
Remove the condition `n === undefined`, use a default parameter for `n` in the definition (`(arr, n = 1)`). Apart from that, this should be fine.

Reply:
Copy, sometimes forget I can do that also. Thanks